langmeta/shared/src/main/scala/org/langmeta/semanticdb/internal/package.scala

```diff
@@ -57,7 +57,7 @@ package object semanticdb {
           }
           object sResolvedSymbol {
             def unapply(sresolvedsymbol: s.ResolvedSymbol): Option[d.ResolvedSymbol] = sresolvedsymbol match {
-              case s.ResolvedSymbol(d.Symbol(dsym), Some(s.Denotation(dflags, dname: String, dsignature: String, snames))) =>
+              case s.ResolvedSymbol(d.Symbol(dsym), Some(s.Denotation(dflags, dname: String, dsignature: String, snames)), members) =>
```

Comment:
Do we want to make any compatibility guarantees about protobuf stubs?

Reply:
No, we provide no compatibility story for generated protobuf code. Users can generate their own code if they want to keep the old `internal.schema.Denotation`. The persisted protobuf messages themselves are backwards and forwards binary compatible, meaning old clients can read messages from the future and new clients can read messages from the past.